spdx-utils/src/main/resources/exceptions/SHL-2.1

```diff
@@ -0,0 +1,45 @@
+
+   "Copyright statement": the reference to 'copyright statement' shall be taken to read 'copyright or other statement pertaining to Rights'
+
+   The following new definition shall be added to the Definitions section of the Apache License:
+
+   "Rights" means copyright and any similar right including design right (whether registered or unregistered), rights in semiconductor topographies (mask works) and database rights (but excluding Patents and Trademarks).
+
+   The following definitions shall replace the corresponding definitions in the Apache License:
+
+   "License" shall mean this Solderpad Hardware License version 2.1, being the terms and conditions for use, manufacture, instantiation, adaptation, reproduction, and distribution as defined by Sections 1 through 9 of this document.
+
+   "Licensor" shall mean the Rights owner or entity authorized by the Rights owner that is granting the License.
+
+   "Derivative Works" shall mean any work, whether in Source or Object form, that is based on (or derived from) the Work and for which the editorial revisions, annotations, elaborations, or other modifications represent, as a whole, an original work of authorship or design. For the purposes of this License, Derivative Works shall not include works that remain reversibly separable from, or merely link (or bind by name) or physically connect to or interoperate with the Work and Derivative Works thereof.
+
+   "Object" form shall mean any form resulting from mechanical transformation or translation of a Source form or the application of a Source form to physical material, including but not limited to compiled object code, generated documentation, the instantiation of a hardware design or physical object or material and conversions to other media types, including intermediate forms such as bytecodes, FPGA bitstreams, moulds, artwork and semiconductor topographies (mask works).
+
+   "Source" form shall mean the preferred form for making modifications, including but not limited to source code, net lists, board layouts, CAD files, documentation source, and configuration files.
+
+   "Work" shall mean the work of authorship or design, whether in Source or Object form, made available under the License, as indicated by a notice relating to Rights that is included in or attached to the work (an example is provided in the Appendix below).
+
+   2. Grant of License. Subject to the terms and conditions of this License, each Contributor hereby grants to You a perpetual, worldwide, non-exclusive, no-charge, royalty-free, irrevocable license under the Rights to reproduce, prepare Derivative Works of, make, adapt, repair, publicly display, publicly perform, sublicense, and distribute the Work and such Derivative Works in Source or Object form and do anything in relation to the Work as if the Rights did not exist. APPENDIX
+
+Copyright [yyyy] [name of copyright owner]
+
+SPDX-License-Identifier: Apache-2.0 WITH SHL-2.1
```

Comment:
Wow, this is interesting, as it uses `WITH` not with an exception, but an addition.

Reply:
See the text is has additions and modifications so it exception plus an addition - have not seen one before ...

Reply:
Anyway, this is the proof that it was a good decision from recently to not requite the part of `WITH` to be a known exception.